examples/oneapi/cpp/source/decision_forest/df_reg_dense_batch.cpp

```diff
@@ -18,45 +18,47 @@
 #include "oneapi/dal/algo/decision_forest.hpp"
 #include "oneapi/dal/io/csv.hpp"
 
-using namespace oneapi;
+namespace onedal = oneapi::dal;
 namespace df = oneapi::dal::decision_forest;
 
-
 int main(int argc, char const *argv[]) {
-  const std::string train_data_file_name  = get_data_path("df_regression_train_data.csv");
-  const std::string train_label_file_name = get_data_path("df_regression_train_label.csv");
-  const std::string test_data_file_name   = get_data_path("df_regression_test_data.csv");
-  const std::string test_label_file_name  = get_data_path("df_regression_test_label.csv");
+    const std::string train_data_file_name = get_data_path("df_regression_train_data.csv");
```

Comment:
```suggestion
    const auto train_data_file_name = get_data_path("df_regression_train_data.csv");
```
just a nip, you don't have to apply changes like this